src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/test/messages/PrisonerSearchMessages.kt

```diff
@@ -0,0 +1,165 @@
+package uk.gov.justice.digital.hmpps.personrecord.test.messages
+
+fun prisonerSearchResponse(nomsNumber: String) = """
```

Comment:
👍 although I think we will find that passing in an object with properties including nomsNumber becomes more useful as we add tests